start-dconf-editor.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/bash
+
+IFS=: read -ra host_data_dirs < <(flatpak-spawn --host sh -c 'echo $XDG_DATA_DIRS')
+
+# To avoid potentially muddying up $XDG_DATA_DIRS too much, we link the schema paths
+# into a temporary directory.
+bridge_dir=$XDG_RUNTIME_DIR/dconf-bridge
+mkdir -p $bridge_dir
+
+for dir in "${host_data_dirs[@]}"; do
+  if [[ "$dir" == /usr/* ]]; then
+    dir=/run/host/"$dir"
+  fi
+
+  schemas="$dir/glib-2.0/schemas"
+  if [[ -d "$schemas" ]]; then
+    bridged=$(mktemp -d XXXXXXXXXX -p $bridge_dir)
+    mkdir -p $bridged/glib-2.0
+    ln -s "$schemas" $bridged/glib-2.0
+    XDG_DATA_DIRS=$XDG_DATA_DIRS:$bridged
```

Comment:
Honestly I am not a fan of this. While it is certainly the most low effort solution `XDG_DATA_DIRS` is full of stuff we absolutely do not want to read.

I believe what would be safer (I've not tested it) is for you to bring everything into a dir, run `glib-compile-schemas` and then set `GSETTINGS_SCHEMA_DIR`.

Reply:
I should probably read the actual script better. I see this limits it to the `glib-2.0` directory which should be good.